.circleci/config.yml

```diff
@@ -55,17 +46,11 @@ jobs:
             fi
           environment:
             JEST_JUNIT_OUTPUT_DIR: ./coverage/junit/
-
       - run:
           name: Integration tests
-          command: |
-            DocId=$(curl -s --data-binary @data/apps/Executive_Dashboard.qvf http://localhost:9076/v1/apps/import | jq -r '.attributes.id')
-            APP_ID=$DocId yarn run test:integration --chrome.browserWSEndpoint "ws://localhost:3000" --no-launch
-
+          command: yarn test:rendering
```

Comment:
Do we want to call it integration tests or rendering tests? If we add actual integration tests, should they be a separate step? If so I think this should be called Rendering tests

Reply:
Right. gonna fix. thanks.

Reply:
Done